docs/using-data.md

```diff
@@ -2,6 +2,22 @@
 
 All of the ClearlyDefined data is available for everyone to see and use. You can browse and inspect in a [convenient web ui](#website) or hook up a client to the [REST API](https://api.clearlydefined.io/api-docs/) and integrate it into your systems.
 
+## API Usage Considerations
+
+Current rate limits are as follows:
+
+### Endpoint-Specific Restrictions
+
+| Endpoint      | Method  | Limit    |
+|---------------|---------|----------|
+| /definitions  | POST    | 250 /min |
+| /curations    | POST    | 250 /min |
+| /notices      | POST    | 250 /min |
```

Comment:
For DEV:
The routes above are correct (250req/ 1min window). For all other routes in dev, the rate is max of 500 requests within a 5 minute window

For PROD:
POSTing to /definitions, /curations, and /notices is max 60 requests per 1 min window. For all other routes its 2k requests per 1 min window

Reply:
@bduranc  i apologize, for PROD, POSTing to the 3 routes is actuallt 250 req / 1 min, not 60/min

Reply:
@mpcen no worries, I've made the changes.

Reply:
@mpcen is there anything else I need to do so we can merge this?

Reply:
I'd suggest including a description of what HTTP response code is given if the rate limit is hit, and when the limit gets reset.

Reply:
Thanks @Keavon for the suggestion.

@qtomlinson : Do you know what these two response codes would be? (_express-rate-limit_ lists a default of HTTP 429, but not sure if that applies here.)

Reply:
I made my latest changes.